css/layouts/_version-nav.scss

```diff
@@ -67,7 +67,7 @@
           content: 'Managed CRDB';
```

Comment:
Can you correct this as well? Just change it to `CRC`?

Reply:
fixed